source/_patterns/04-components/menu/menu--account/menu--account.md

```diff
@@ -0,0 +1,23 @@
+---
+el: .menu--account
+title: Account Menu
+---
+
+__Variables:__
+* menu_name: [string] Machine name of the menu.
+* menu_class: [string] Optional classes to add to the menu.
+* attributes: [array] Menu attributes.
+* items: [array] Menu items. Each item is an object containing:
+  * title: [string] Title of the menu item.
+  * url: [string] URL of the menu item.
+  * original_link: [array] Options and HTML attributes for the menu link item.
+  * in_active_trail: [boolean] Whether menu item is in the active path.
+
+__Usage:__
+The menu components folder contains several variants including _Account menu_,
+_Footer menu_, _Main menu_ and _Menu with subnav_.  Each variant is derived from
```

Comment:
Super minor, but since this PR is focused on formatting: Seems like the component names should be in title case here ("Account Menu") since that's how the title for each component is written. Same goes for the other .md files where this text appears.